lib/health_cards/exporter.rb

```diff
@@ -14,14 +14,39 @@ def download(jws)
       # Export JWS for $issue-health-card endpoint
       # @param [Array<JWS>, String] An array of JWS objects to be exported
       # @return [String] JSON string containing a FHIR Parameters resource
-      def issue(jws)
-        params = jws.map { |j| FHIR::Parameters::Parameter.new(name: 'verifiableCredential', valueString: j) }
+      def issue(fhir_params)
```

Comment:
The documentation for this method needs updating. It doesn't look like it takes a string anymore.

Reply:
fixed